views/events/new_zealand.erb

```diff
@@ -0,0 +1,42 @@
+<h2 id="nz_february_2017">New Zealand, Mount Cheeseman, February 2017</h2>
+<dl>
+  <dt>When</dt>
+  <dd>
+    <p>Starts on the afternoon of Friday February 24th, and runs until midday on Monday 27th.</p>
+  </dd>
+  <dt>Where</dt>
+  <dd>
+    <p>At the <a href="http://www.mtcheeseman.co.nz/">Mt Cheeseman Ski Lodge</a> (<a href="https://maps.google.com/maps?ll=-43.156921,171.668415&z=8&cid=17347277282273374815">map</a>), near Christchurch, New Zealand.</p>
+  </dd>
+  <dt>Can Haz?</dt>
+  <dd>
+    <p>Please do come along! Tickets are <a href="https://ti.to/rails-camp-nz/2017">available now</a> for NZ$289.</p>
+  </dd>
+  <dt>What to Bring</dt>
+  <dd>
+    <ul>
+      <li class="first">Clothes</li>
+      <li>Laptop</li>
+      <li>Camera</li>
+      <li>Chargers</li>
+      <li>Powerboard</li>
+      <li>Toothbrush &amp; Toiletries</li>
+      <li>Towel</li>
+      <li>Sleeping Bag</li>
+      <li>Pillowcase</li>
+      <li>Earplugs</li>
+      <li>Your alcoholic beverages of choice</li>
```

Comment:
I think it's worth mentioning whether alcohol is included or not.
Maybe this can be changed to:
> Alcohol is not included in the ticket price, so if you wish to drink then pack these too